qml/pages/VenueList.qml

```diff
@@ -155,6 +155,7 @@ Page {
                     leftMargin: Theme.horizontalPageMargin
                     rightMargin: Theme.horizontalPageMargin
 
+                    topMargin: Theme.paddingSmall
```

Comment:
I don't think that it worsens it, but why is the distance base-aligned to the street? That's a bug right?